### PR TITLE
Signup login page

### PR DESCRIPTION
**Pull request by VedanthB**
- added signup page
- added login page

### CODE REVIEW

**Review by ishapatell1 — APPROVED** [review]

**Review by ishapatell1 — APPROVED** [review]
Looks good and clean. Please make it a point to add snapshots, it becomes easier to see the UI design.

File: pages/product-page/product-listing.css [hunk 1]
```diff
@@ -0,0 +1,15 @@
+.divider {
+  border: 1px solid var(--regal-blue-dark);
+  width: 80%;
+  margin: auto;
+}
+.filters-sidebar-container {
+  height: 100%;
+  width: 20%;
+  /* border: 1px solid var(--regal-blue-dark); */
```
Comment (dev-enforced) [inline pages/product-page/product-listing.css:9]: Can remove this commented line of code

**Review by sharathnair9999 — APPROVED** [review]

**Review by sharathnair9999 — APPROVED** [review]

**Review by dev-enforced — APPROVED** [review]
The code looks good to me!! Do split your files into smaller ones so that it becomes easy to review 
And of course the screenshot point given by @Isha2806 is also important to follow further
Good work @VedanthB !!

File: pages/product-page/product-listing.html [hunk 2]
```diff
@@ -78,9 +78,9 @@ <h6 class="h6 cursor-pointer text-white text-hover-cyan-500"> Shop </h6>
     <!-- header -->
 
     <!-- main section -->
-    <main style="top: 5rem; min-height: 100vh" class="relative">
+    <main style="top: 5rem; min-height: 100vh" class="relative w-full justify-end flex">
         <!-- filters -->
-        <div style="top: 8rem" class="filters-sidebar-container relative ml-10 p-5 ">
+        <div style="top: 8rem" class="filters-sidebar-container relative ml-10 p-5">
```
Comment (sharathnair9999) [inline pages/product-page/product-listing.html:89]: you can avoid inline styling.

**Review by sharathnair9999 — APPROVED** [review]

File: pages/product-page/product-listing.html (hunk 2 above)
Comment (VedanthB) [reply pages/product-page/product-listing.html:89]: Okay will keep that in mind

**Pull request merged**